config/test_glyphs-glyf_colr_1.py

```diff
@@ -67,319 +69,364 @@ def _deltaOrZero(axis: str, position: dict):
     return position.get(axis, 0)
 
 
-def _sample_sweep(
-    start_angle, end_angle, extend_mode_arg, color_line_range, position, accessor
-):
+class TestCategory(ABC):
+    def __init__(self, range_start, range_end):
+        # Used for backwards-compatibility when glyphs are updated or changed. Subclasses should add glyphs here when
+        # they are redefined or replaced with a different meaning.
+        self.burned_codepoint_offsets = []
+        self.range_start = range_start
+        self.range_end = range_end
 
-    extend_mode_map = {
-        "reflect": ot.ExtendMode.REFLECT,
-        "repeat": ot.ExtendMode.REPEAT,
-        "pad": ot.ExtendMode.PAD,
-    }
+    @abstractmethod
+    def get_name(self) -> str:
+        return ""
 
-    if extend_mode_arg not in extend_mode_map:
-        return None
+    def make_test_glyphs(self, position) -> list[SampleGlyph]:
+        accessors = iter(self._get_accessors())
+        for param_set in self._get_test_parameters():
```

Comment:
effectively this is like doing

```python
for accessor, param_set in zip(self._get_accessors(), self._get_test_parameters())
    yield self._make_test_glyph(param_set, position, accessor)
```

`zip()` (unless strict=True) stops when the shortest iterable is exhausted, which in your case is `self._get_test_parameters()` if I am not mistaken, and it's what you want it to do.